THIRD_PARTY_NOTICES.md

```
@@ -8,3 +8,13 @@ This project includes derived English definitions in `data/dictionaries/en-defin
 
 - WordNet license text: `data/dictionaries/wordnet-3.1-LICENSE.txt`
 - Generator script: `scripts/build-en-definitions.js`
+
+## Chart.js
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)